test/property.test.js

```diff
@@ -83,7 +152,103 @@ describe('loopback:property generator', function() {
     });
   });
   
-  it('creates a defaultFn: "now" on date fields if specified', function(done) {
+  it('creates string item typed array', function(done) {
+    var propertyGenerator = givenPropertyGenerator();
+    helpers.mockPrompt(propertyGenerator, {
+      model: 'Car',
+      name: 'options',
+      type: 'array',
+      itemType: 'string',
+      required: 'true',
+      defaultValue: 'AWD,3.2L, navigation'
+    });
+
+    propertyGenerator.run(function() {
+      var definition = common.readJsonSync('common/models/car.json');
+      var props = definition.properties || {};
+      expect(props).to.have.property('options');
+      expect(props.options).to.eql({
+        type: 'array',
+        itemType: 'string',
```

Comment:
Not sure where does the excerpt above is aimed, but given that `mockPrompt` doesn't accept string typed array, it's probably related to the expected type of the options param.